metals/src/main/scala/scala/meta/internal/metals/WorkspaceLspService.scala

```diff
@@ -676,6 +676,11 @@ class WorkspaceLspService(
         getServiceFor(unresolvedParams.path)
           .discoverMainClasses(unresolvedParams)
           .asJavaObject
+      case ServerCommands.ResetWorkspace() =>
+        onCurrentFolder(
```

Comment:
We should do it for all projects if we name it `ResetWorkspace` and I think that makes sense